.gitignore

```diff
@@ -6,3 +6,6 @@
 /wordpress
 /wp-content
 vendor
+
+# GitHub Action related files and directories
+/translation-action-scripts
```

Comment:
@mr-vara  As discussed, let's remove this in favor of storing temporary files in a subfolder in the `{{ $runner.temp }}` directory.